CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.1.3] - 2024/11/..
```

Comment:
Thanks Kyla. Can you add a link at the bottom of the page here to the expected tag for when this version is released?

Reply:
Change made here: [a20b157](https://github.com/phac-nml/genomic_address_service/pull/14/commits/a20b15751ecbee0770a563e5c293c00ef78cfb39)